<map id="pymongo.errors.ConfigurationError" name="pymongo.errors.ConfigurationError">
<area shape="rect" id="node4" href="$classpymongo_1_1errors_1_1_invalid_u_r_i.html" title="pymongo.errors.InvalidURI" alt="" coords="19,244,191,271"/>
<area shape="rect" id="node2" href="$classpymongo_1_1errors_1_1_py_mongo_error.html" title="pymongo.errors.PyMongoError" alt="" coords="5,80,204,107"/>
</map>
